pkg/source/ttn/config.go

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ttn
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"os"
+
+	ttnsdk "github.com/TheThingsNetwork/go-app-sdk"
+	ttnlog "github.com/TheThingsNetwork/go-utils/log"
+	"github.com/TheThingsNetwork/go-utils/log/apex"
+	"github.com/spf13/pflag"
+)
+
+const (
+	clientName = "ttn-lw-migrate"
+)
+
+type config struct {
+	sdkConfig ttnsdk.ClientConfig
+
+	debug bool
+
+	appAccessKey string
+	appID        string
+
+	frequencyPlanID   string
+	withFrameCounters bool
+}
+
+func flagSet() *pflag.FlagSet {
+	flags := &pflag.FlagSet{}
+	flags.Bool("ttn.without-frame-counters", false, "Do not export device frame counters (faster)")
```

Comment:
I still don't understand why you want to migrate devices faster in terms of time, but leaving the door open to replay attacks and potentially breaking downlink because the FCntDown is too low.